src/cms/src/api/site/content-types/site/schema.json

```diff
@@ -58,7 +58,8 @@
       "type": "relation",
       "relation": "manyToOne",
       "target": "api::protected-area.protected-area",
-      "inversedBy": "sites"
+      "inversedBy": "sites",
+      "required": true
```

Comment:
Does this `required` attribute actually do anything?  I noticed you don't have one for park-sub-pages

Reply:
I have `required` for `site` and `park-sub-page` 🙂 
It just displays `*` on Strapi UI to indicate it's a required field.

![Screenshot 2023-05-30 at 9 35 20 AM](https://github.com/bcgov/bcparks.ca/assets/90415568/560464a7-04a0-4bcd-bf46-e844f1f0f4d5)